CommonTools/CandAlgos/plugins/CandPtrProjector.cc

```diff
@@ -53,13 +53,13 @@ CandPtrProjector::produce(edm::StreamID, edm::Event& iEvent, edm::EventSetup con
     if (vetoedPtrs.find(c)==vetoedPtrs.cend()) {
       bool addcand = true;
       if (useDeltaRforFootprint_)
-	for (const auto& it : vetoedPtrs)
-	  if ((it.isNonnull()) && (it.isAvailable()) && (reco::deltaR2(it->p4(), c->p4()) < 0.00000025)) {
+        for (const auto& it : vetoedPtrs)
+	  if (it.isNonnull() && it.isAvailable() && reco::deltaR2(it->p4(), c->p4()) < 0.00000025) {
```

Comment:
```suggestion
          if (it.isNonnull() && it.isAvailable() && reco::deltaR2(*it, *c) < 0.00000025) {
```
- a case of a tab showing a bad indentation for a diff
- catch up with CPU use fixes, as in #29919 (106X)